lib/Util/SVFUtil.cpp

```diff
@@ -306,7 +306,7 @@ std::string SVFUtil::getSourceLoc(const Value* val)
             rawstr << "ln: " << Line << "  cl: " << Column << "  fl: " << File;
         }
     }
-    else if (const Argument* argument = SVFUtil::dyn_cast<Argument>(val))
```

Comment:
Ideally, this should be `const SVFArgument* argument = SVFUtil::dyn_cast<Argument>(SVFValue)` later.